src/exec/Shelley.hs

```diff
@@ -104,3 +109,9 @@ mkParams rr cdb nconf nstate blockchainTime = NodeArgs
   , maxUnackTxs = maxBound
   , chainSyncPipelining = pipelineDecisionLowHighMark 200 300 -- TODO: make configurable!
   }
+
+--
+-- Orphans
+--
+
+deriving via OnlyCheckIsWHNF "Peer" Peer instance NoUnexpectedThunks Peer 
```

Comment:
Shouldn't this instance be with the `Peer` type? The `cardano-node` needs this same instance.

Reply:
`Peer` is defined in this module; it's not an orphan.

Reply:
If its is defined in this module then the instance should be moved to the type declaration.

Reply:
Entirely possible. This was the end of a long day and I just wanted to try it and see if any changes were needed.